src/routers/handlers/generic.ts

```diff
@@ -1,17 +1,38 @@
 // Generic handler is the base handler that is extended by all other handlers
 // It contains methods that are common to multiple route handlers
 
-import errorManifest from "./../../lib/utils/error_manifests/default";
+import errorManifest from "../../lib/utils/error_manifests/default";
+import { Request } from "express";
 
-export class GenericHandler {
+export interface BaseViewData {
+    errors: any
+    title: string
+    isSignedIn: boolean
+    backURL: string | null
+    sampleKey?: string
```

Comment:
What is this variable? Is it needed?

Reply:
thought of using a string[] to hold placeholder values but can take it out